qemu/tests/cfg/dump_guest_core.cfg

```diff
@@ -0,0 +1,20 @@
+- dump_guest_core:
+    type = dump_guest_core
+    virt_test_type = qemu
+    start_vm = no
+    pre_command = "ulimit -c unlimited"
+    pre_command1 = "echo "/var/core.%p" > /proc/sys/kernel/core_pattern"
+    trigger_core_dump_command = "kill -s SIGSEGV %s"
+    gdb_command = "gdb /var/core.%s --command=%s"
+    gdb_command_file = "/home/gdb_command"
+    crash_script = "/home/crash.cmd"
+    vmcore_file = "/tmp/vmcore"
+    extra_params += " -machine dump-guest-core=%s"
```

Comment:
You might move this line down to the bottom and `extra_params += " -machine dump-guest-core=${dump_guest_core}"`